example/package.json

```diff
@@ -12,6 +12,9 @@
   },
   "devDependencies": {
     "@babel/core": "^7.0.0",
+    "@react-native-community/cli": "^3.0.0",
```

Comment:
What was the reason to bump the version?

Reply:
2.x doesn't support this folder structure. It fails to find `react-native`.